string name
string addr
uint8 strength
uint8 strength_total
int8 signal_level
uint8 frequency
string last_check
